bingo-cards-ui/src/app/app.component.ts

```diff
@@ -27,11 +30,23 @@ export class AppComponent implements OnInit {
     const c: ICreateMeeting = { userName: '', meetingName: '' };
     this.createMeetingForm = this.formBuilder.group(c, Validators.required);
   }
+  // _nameParticipant = this.createMeetingForm.get('userName').value;
+  get _nameParticipant(): any {
+    return this.createMeetingForm.get('userName').value;
+  }
   getMeetingValue(id) {
     console.log(this.meetings);
     this.mName = id;
   }
+  addParticipant(idMeeting: string) {
+    console.log(idMeeting);
+    const participant = {
+      meetingId: idMeeting,
+      nameParticipant: this._nameParticipant
+    };
 
+    this.cardsService.addParticipant(participant).subscribe(data => console.log(data));
```

Comment:
Should make a message about the success or the error  of the operation

Reply:
ok, working on it